### PR TITLE
fix cp2k basis parse (issue #2043)

### PR DESCRIPTION
**Pull request by fishjojo**
TODO: allow choosing symbols when parsing GTH PPs.

### CODE REVIEW

**Comment by verena-neufeld** [comment]
Thanks @fishjojo . Since tests didn't pick this bug up, do you think we need new tests for this?

File: pyscf/gto/basis/parse_cp2k_pp.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 from pyscf.lib.exceptions import BasisNotFoundError
 import numpy as np
 
-def parse(string):
+def parse(string, symb=None):
```
Comment (verena-neufeld) [inline pyscf/gto/basis/parse_cp2k_pp.py:27]: I don't think the `symb` keyword is used in that function. Is that there for consistency? Should individual element's PP be parsed?

**Comment by verena-neufeld** [comment]
Thanks so much @fishjojo ! I think it looks great. I've only commented with one more small question (sorry, I should have done that yesterday with the other question).

File: pyscf/gto/basis/parse_cp2k_pp.py (hunk 1 above)
Comment (sunqm) [reply pyscf/gto/basis/parse_cp2k_pp.py:27]: I guess this is just to make parse_cp2k_pp function signatures the same to that in parse_nwchem.

**Review by sunqm — APPROVED** [review]

File: pyscf/gto/basis/parse_cp2k_pp.py (hunk 1 above)
Comment (verena-neufeld) [reply pyscf/gto/basis/parse_cp2k_pp.py:27]: Ok, thanks!

Comment (fishjojo) [reply pyscf/gto/basis/parse_cp2k_pp.py:27]: I can also add symbol choosing for PPs, maybe later this week.

**Pull request merged**